exercises/transpose/example.js

```diff
@@ -0,0 +1,12 @@
+export default function transpose(text) {
+  const result = [];
+  text.forEach((line, lineNo) => {
+    line.split('').forEach((value, key) => {
```

Comment:
```javascript
   [...line].forEach((value, key) => {
```

The `...` is the "spread" operator.  [It's awesome](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_operator).